src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -72,9 +74,12 @@ export const RichTextEditor: FC<RichTextEditorProps> = ({
     const [{ matches, children }, send] = useMachine(() =>
         editorMachine.withContext({ locked: readonly, onTextChange, onBlur }),
     );
-    useEditorValueUpdates(editor, initialValue);
 
     useEffect(() => {
+        if (clear !== undefined) {
```

Comment:
you can put it into its own useEffect, since it should be executed every time the property changes:
```
useEffect(() => {
    clear && clearEditor(editor);
, [clear]}
```

(with clear as boolean)

Reply:
Yes, makes sense 👍  I wasn't sure if you may want to execute something in the block settings before clearing